src/main/java/com/datalogics/pdf/samples/signature/SignDocument.java

```diff
@@ -169,6 +183,10 @@ private static void signField(final SignatureManager sigMgr,
 
                     appearanceOptions.setFontSet(fontSet);
                     appearanceOptions.setDisplayItems(displayItems);
+
+                    // Customize the label. See the declaration of the constant for more info.
+                    appearanceOptions.setSignatureLabel(SIGNATURE_LABEL);
```

Comment:
Eclipse can't find this recently added 'setSignatureLabel'. I think you need to exclude this from the Talkeetna 5 build like we did with the export to csv sample.